dot/rpc/modules/state.go

```diff
@@ -121,8 +120,10 @@ type StateGetReadProofResponse struct {
 
 // StorageChangeSetResponse is the struct that holds the block and changes
 type StorageChangeSetResponse struct {
-	Block   *common.Hash `json:"block"`
-	Changes [][]string   `json:"changes"`
+	Block *common.Hash `json:"block"`
+	// Changes is a slice of string pointers so that the JSON encoder can handle nil values as NULL instead of empty
+	//  strings.
```

Comment:
nit moaare splitting of comment lines and sneaking it `array` in there

```suggestion
	// Changes is a slice of arrays of string pointers instead of just strings
	// so that the JSON encoder can handle nil values as NULL instead of empty
	// strings.
```